final_prep/annotations.rb

```diff
@@ -16,28 +20,39 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+# Return function
   return built_bear
 end
 
+# Run function build_a_bear
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
-
+# Define function with three arguments
 def fizzbuzz(num_1, num_2, range)
+  # Run for look for range argument
   (1..range).each do |i|
+    # Define if statement
     if i % num_1 === 0 && i % num_2 === 0
+      # Print string if statment is true
       puts 'fizzbuzz'
+    # Define elsif statment
     elsif i % num_1 === 0
+      # Print string if elsif is true
       puts 'fizz'
+    # Define elsif statment
     elsif i % num_2 === 0
+      # Print string if elsif is true
       puts 'buzz'
+    # Define elsif statment
     else
+      # Print i
```

Comment:
I would recommend (at least in the beginning) striving to be as detailed as you can in your annotations - it will really help demonstrate (to yourself and others) how well you grasp technical concepts and terminology.